svc-bip-api/src/main/resources/application.properties

```diff
@@ -0,0 +1,49 @@
+# FIXME: Why is this needed?
+server.port=8091
+# This file contains shared properties across all environments; it is always loaded by Spring
+# See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings#vros-use-of-spring-profiles
+
+spring.profiles.active=${ENV:default}
+
+## Define Sprint Profile Groups with group names corresponding to the value of $ENV
+# Configure profile group so that application-nonprod.properties is loaded,
+# along with application-$ENV.properties, if it exists
+spring.profiles.group.local=nonprod
+# and integrationTest
+spring.profiles.group.integrationTest=nonprod
+# Remaining profiles are for deployment to LHDI's EKS environments
+# Configure profile group so that application-nonprod.properties and application-k8s.properties is loaded,
+# along with application-$ENV.properties, if it exists
+spring.profiles.group.dev=nonprod,k8s
+spring.profiles.group.qa=nonprod,k8s
+spring.profiles.group.sandbox=nonprod,k8s
+# For prod, only load application-prod.properties and application-k8s.properties, if they exist
+spring.profiles.group.prod=k8s
+
+## RabbitMQ Settings
+
+spring.rabbitmq.host=${RABBITMQ_PLACEHOLDERS_HOST}
+spring.rabbitmq.port=5672
+spring.rabbitmq.username=${RABBITMQ_PLACEHOLDERS_USERNAME}
+spring.rabbitmq.password=${RABBITMQ_PLACEHOLDERS_USERPASSWORD}
+
+abd-data-access.rabbitmq.claim-submit-exchange=claim-submit-exchange
+abd-data-access.rabbitmq.claim-submit-queue=claim-submit
+abd-data-access.rabbitmq.claim-submit-routing-key=code.#
```

Comment:
`abd-data-access` was/is used for the `svc-lighthouse-api` client. Use something else like `bip-api-client`.